department-of-veterans-affairs/caseflow
# Update Appeal type label on claim details page

Resolves #15020

### Description
Update appeal type label in COPY.json 

### Acceptance Criteria
- [ ] Type label for appeal type says 'Appeal Stream Type' on Claim Details page 

### Testing Plan
1. Choose user with edit hearing details permissions
2. Click Mail Intake tab
3. Click a hearing day 
4. Click the ID link underneath a veteran's name 
5. Verify that the case title details header now includes 'Appeal Stream Type'

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="1069" alt="Screen Shot 2020-09-02 at 3 52 00 PM" src="https://user-images.githubusercontent.com/68879427/92046341-2a7c6100-ed37-11ea-861a-8a1c96721806.png"> | <img width="1064" alt="Screen Shot 2020-09-02 at 3 53 51 PM" src="https://user-images.githubusercontent.com/68879427/92046355-2f411500-ed37-11ea-8cd3-5ce934290e6c.png">


### Storybook Story
*For Frontend (Presentationa) Components*
